src/test/groovy/com/bisnode/opa/client/QueryingForDocumentSpec.groovy

```diff
@@ -141,10 +141,9 @@ class QueryingForDocumentSpec extends Specification {
         when:
 
             def type = TypeUtils.parameterize(List.class,ValidationResult.class);
-            def result = client.queryForDocument(new QueryForDocumentRequest([shouldPass: true], path), type)
+            List<ValidationResult> result = client.queryForDocument(new QueryForDocumentRequest([shouldPass: true], path), type)
         then:
-            def element = result[0]
-            element.getClass() == ValidationResult.class
```

Comment:
Not sure if removing this is good idea. Whole point of this test was to check if the class was mapped correctly.